sdk/identity/azure-identity/README.md

```diff
@@ -274,14 +250,39 @@ public void createUserNamePasswordCredential() {
 }
 ```
 
+### Authenticating a user account interactively in the browser
+This example demonstrates authenticating the `KeyClient` from the [azure-security-keyvault-keys][keys_client_library] client library using the `InteractiveBrowserCredential`. There's also [a compilable sample](../../keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java) to create a Key Vault secret client you can copy-paste. 
+
+See more about how to configure an AAD application for interactive browser authentication and listen on a port locally in [Enable applications for interactive browser oauth 2 flow](#enable-applications-for-interactive-browser-oauth-2-flow)
+
+<!-- embedme ../../keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java#L190-L203 -->
+```java
+/**
+ * Authenticate interactively in the browser.
+ */
+public void createInteractiveBrowserCredential() {
+    InteractiveBrowserCredential interactiveBrowserCredential = new InteractiveBrowserCredentialBuilder()
+        .port(8765)
```

Comment:
Client Id is required for Interactive Browser Cred, so this won't work as it is.

Reply:
added